api/proto/config/managed_delivery.proto

```diff
@@ -0,0 +1,113 @@
+syntax = "proto3";
+package proto.config;
+
+option go_package = "github.com/spinnaker/kleat/api/client/config";
+
+import "google/protobuf/wrappers.proto";
+import "storage/sql.proto";
+
+message ManagedDelivery {
+  // Whether ManagedDelivery is enabled.
+    google.protobuf.BoolValue enabled = 1;
+    // Service discovery done through Eureka
+    Eureka eureka = 2;
+    // Keel specific configuration
+    KeelConfig keel = 3;
+
+}
+
+message Keel {
+    // Keel specific configuration
+    KeelConfig keel = 1;
+
+    // Service discovery done through Eureka
+    Eureka eureka = 2;
+
+    // Datastore used for keeping Keel data
+    storage.SQL sql = 3;
+}
+
+message Eureka {
+    // Whether this persistent store is enabled.
+    google.protobuf.BoolValue enabled = 1;
+}
+
+message KeelConfig {
+    // Managed Delivery plugins to be enabled
+    Plugins plugins = 1;
+    // duration string format (eg. 10s)
```

Comment:
This and the next comment should also briefly describe the field (as they are used in documentation).